changelogs/fragments/keycloak_realm_ssl_required.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - keycloak_realm module plugin - 'ssl_required' changed from a bool type to accept 'none', 'external' or 'all' (https://github.com/ansible-collections/community.general/pull/2693).
```

This is how this entry would be acceptable. If the later part is wrong (about always failing), then we have to do this whole PR differently.
```suggestion
  - keycloak_realm - ``ssl_required`` changed from a boolean type to accept the strings ``none``, ``external`` or ``all``. This is not a breaking change since the module always failed when a boolean was supplied (https://github.com/ansible-collections/community.general/pull/2693).
```

---

Thanks, I've applied you suggestion, this is afaict true, I could not get a `true` or `false` statement pass here, and would get 500s from the server when trying and thus should not be a breaking change.